sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -57,6 +57,7 @@ floating-point type(`float`) to `bfloat16` type and vice versa. The extension
 doesn't add support for `bfloat16` type as such, instead it uses 16-bit integer
 type(`uint16_t`) as a storage for `bfloat16` values.
 
+
```

Comment:
I  assume this is somehow inadvertent?  You could remove this blank line and then you won't need a reviewer from @dpcpp-specification-reviewers.

Reply:
Hi yes sorry I'm not sure how that crept in. Have removed.